src/connectors/hits-per-page/connectHitsPerPage.js

```diff
@@ -115,6 +117,13 @@ export default function connectHitsPerPage(renderFn) {
     }
 
     return {
+      getConfiguration() {
+        const { value: defaultHitsPerPage } =
+          items.find(item => item.default) || items[0];
+
+        return { hitsPerPage: defaultHitsPerPage };
```

Comment:
One thing that could be great is to display an error message when there are more than one `default:true`

Reply:
👍